tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "target": "ES2020",
     "module": "ES2020",
     "moduleResolution": "Node",
+    "jsx": "react",
```

Comment:
Is this really needed? I don't think generated rules use that

Reply:
I highly assume its not
But I will check tomorrow

Reply:
Removed